examples/jsm/objects/LightningStorm.d.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Material,
+  Vector3
+} from '../../../src/Three';
+
+export interface StormParams {
+  size: number | undefined;
+  minHeight: number | undefined;
+  maxHeight: number | undefined;
+  maxSlope: number | undefined;
+
+  maxLightnings: number | undefined;
+
+  lightningMinPeriod: number | undefined;
+  lightningMaxPeriod: number | undefined;
+  lightningMinDuration: number | undefined;
+  lightningMaxDuration: number | undefined;
+
+  lightningParameters: RayParameters | undefined;
+  lightningMaterial: Material | undefined;
+
+  isEternal: boolean | undefined;
+
+  onRayPosition?: (source: Vector3, dest: Vector3) => void;
+  onLightningDown?: (lightning: LightningStrike) => void;
```

Comment:
`LightningStrike` is also undefined. You have to import the type like so:

```js
import { LightningStrike } from '../geometries/LightningStrike';
```

Reply:
Done. Thanks!